eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -237,7 +237,7 @@ stages:
       - template: /eng/pipelines/templates/variables/globals.yml
     jobs:
       - ${{ each artifact in parameters.Artifacts }}:
-          - ${{if and(ne(artifact.skipPublishDevFeed, 'true'), or(eq(variables['SetDevVersion'], 'true'), and(eq(variables['Build.Reason'],'Schedule'), eq(variables['System.TeamProject'], 'internal'))))}}:
+          - ${{if and(ne(artifact.skipPublishDevFeed, 'true'), or(eq(variables['SetDevVersion'], 'true'), and(true, eq(variables['System.TeamProject'], 'internal'))))}}:
```

Comment:
why are we passing the literal `true` to the `and()` function?

Reply:
this is for faster testing, will remove before merging